disnake/ext/commands/bot.py

```diff
@@ -216,9 +233,7 @@ def __init__(
             owner_ids: Optional[Set[int]] = None,
             reload: bool = False,
             case_insensitive: bool = False,
-            sync_commands: bool = True,
-            sync_commands_debug: bool = False,
-            sync_commands_on_cog_unload: bool = True,
```

Comment:
if I do that I would have to make two overloads

Reply:
It would probably be fine to just keep both in the same definition, the annotations don't have to be 100% accurate for something like this, as long as it still errors early at runtime.
I'm alright with keeping it like this though, two overloads wouldn't be a great alternative.

Reply:
https://github.com/DisnakeDev/disnake/pull/806/commits/981057f20ba372c951586ddbd4891bafdb651982